tools/bundle/static_file_generator.sh

```diff
@@ -3,6 +3,8 @@
 CONFIG_FILE=$1
 BUILD_TYPE=$2
 
+sh ./tools/setup/setup_hooks.sh
```

Comment:
I think hooks could be advice as part of the documentation, and as mandatory for collaborators. But an static file generator shouldn't register hooks in the user's environment.